content/docs/operations/endpoints.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Network Endpoints
+weight: 20
+---
+
+# Exposed Services
```

Comment:
I believe the `title: Network Endpoints` becomes an `<h1>`, so we shouldn't need another header right at the top.

![image](https://user-images.githubusercontent.com/4210949/118701450-bb21a100-b7e1-11eb-9ffe-c772e702a4d3.png)